deploy/charts/trust-manager/values.yaml

```diff
@@ -227,6 +227,11 @@ app:
       # nodePort: 8080
 
     tls:
+      # Whether to issue an "isolated" cert, which removes the requirement for cert-manager to be installed by using Helm to issue a webhook certificate.
+      # Helm-issued certificates can be challenging to rotate and maintain. The issued cert will have a duration of 10 years.
+      # It's safer and easier to rely on cert-manager for issuing the webhook cert.
+      issueIsolatedCert: false
```

Comment:
The term "isolated cert" confuses me. How is it more isolated than cert-manager certificates? Naming is hard, but I would personally be happier with something in the direction of "staticCert".

Reply:
As we discussed in standup this morning, I'll rename this to `helmCert.enabled`!